builder/prepare-ubuntu.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+#
+# This shell script is meant to prepare a building/exec environment for libbpfgo.
+#
+
+die() {
+    echo "ERROR: ${*}"
+    exit 1
+}
+
+info() {
+    echo "INFO: ${*}"
+}
+
+USER="${1}"
+[ -z "${USER}" ] && USER="root"
+USERHOME=$(eval echo ~"${USER}")
+[ -d "${USERHOME}" ] || die "directory ${USERHOME} doesn't exist"
+
+[ -z "${GOVERSION}" ] && GOVERSION="1.17.13"
+[ -z "${ARCH}" ] && ARCH="amd64"
+
+CMDS="wget rm tar chown apt-get"
+for cmd in ${CMDS}; do
+    command -v "${cmd}" >/dev/null 2>&1 || die "missing required tool ${cmd}"
+done
+
+install_go() {
+    info "Installing Go ${GOVERSION}"
+
+    local file="go${GOVERSION}.linux-${ARCH}.tar.gz"
+    local link="https://golang.org/dl/${file}"
```

Comment:
Ubuntu has all golang versions supported (integrated to the OS) and patched (golang-1.17-go, golang-1.18-go, golang-1.19-go) and 1 is the default (golang-go). Can't you rely on the default repository ? Then you can simply "update-alternatives" to /usr/bin/go and /usr/bin/gopls to the version you want (dpkg -L golang-1.17-go | grep bin).

Reply:
At start I thought having it as an option since we have problems with ubuntu not providing specific tooling versions. But for this case, I assume you're right. I'll check the 22.04 package lists and change it accordingly.